Sources/TidepoolKit/TAPI.swift

```diff
@@ -296,35 +232,47 @@ public actor TAPI {
 
         let config = try await lookupOIDConfiguration(environment: session.environment)
 
-        guard let oidConfig = OIDServiceConfiguration(from: config) else {
+        var components = URLComponents()
+        components.queryItems = [
+            URLQueryItem(name:"refresh_token", value: refreshToken),
+            URLQueryItem(name:"grant_type", value: "refresh_token"),
```

Comment:
Just checking this is correct?

Reply:
Are you asking about the `grant_type`?  If so, yes, "refresh_token" is the correct grant type when asking to refresh the access token.